### ISSUE TITLE
Must 'has soma location' be range-restricted to 'connected anatomical structure'?

### ISSUE THREAD

**Issue by gouttegd** [oborel/obo-relations#702]
_has soma location_ (RO:0002100) has a range restriction to _connected anatomical structure_ (CARO:0000003).

Combined with PATO’s general class axiom enforcing a disjunction between `'has characteristic' some 'maximally connected'` and `'has characteristic' some disconnected`, this prevents any neuron from having its soma located in any kind of anatomical structure that is not _maximally connected_.

When merging FBbt with Uberon/CL, this causes ~600 unsatisfiable classes due to a large number of fly neurons that are said in FBbt to have their soma locations in some kind of sensory clusters, because _anatomical clusters_ are **disconnected** entities.

Can the range restriction of _has soma location_ be relaxed to a _material anatomical entity_ with no assertions regarding its “connectedness”?

If not, what would be the appropriate relation to state that a neuron has its soma within a _disconnected_ structure such as a cluster? _part of_ is not ideal since the neuron may not _entirely_ be located within the cluster – its neurites may extend beyond.

**Comment by dosumis** [comment]
> Can the range restriction of has soma location be relaxed to a material anatomical entity with no assertions regarding its “connectedness”?

Yes please.  I think this is also incompatible with the property chain over part_of as connected structures can be part of disconnected ones.

**Comment by cmungall** [comment]
agreed and in general I'd like to review any D/R restrictions that are to abstract classes like this
